### DISCUSSION TITLE
Collection of Issues to Make Flutter Web More Stable / Reliable

### DISCUSSION THREAD

**Discussion by jezell** [join-the-flock/nest#31] [Ideas]
Here are some issues I'd like to fix as soon as the monorepo merge happens on the main branch and we can start doing engine work:

Web: Move decodeImageFromPixels off heap / drastically reduce memory usage
https://github.com/flutter/flutter/issues/153184

WebGPU Backend for Flutter Web
https://github.com/flutter/flutter/issues/124334

RawImage Leaks when the image is changed
https://github.com/flutter/flutter/issues/159902

Infinite Loops and Hangs on CanvasKit Web
https://github.com/flutter/flutter/issues/145380

Flutter CanvasKit Leaks Everything for Every Hot Reload
https://github.com/flutter/flutter/issues/145645

It's still way too easy to run out of platform overlays on web
https://github.com/flutter/flutter/issues/144590

Higher level video support for web 
https://github.com/flutter/flutter/issues/150592

**Comment by clotodex** [comment]
A few things I am following, sorted by priority decreasing for me:
https://github.com/flutter/flutter/issues/124205 - a HUGELY annoying bug when focusing text fields on (mobile) browsers, making the browser offset a good chunk of the app offscreen
https://github.com/flutter/flutter/issues/69529 - communication of scroll events between flutter and browser
https://github.com/flutter/flutter/issues/67915 - rawkeyboardlistener and web are not friends and very buggy (focus is the cause)
https://github.com/flutter/flutter/issues/65170 & https://github.com/flutter/flutter/issues/86838 - more key event issues - enter specifically
https://github.com/flutter/flutter/issues/120613 - text editing overhaul as a whole
https://github.com/flutter/flutter/issues/135159 & https://github.com/flutter/flutter/issues/124483 - alignment issues with textfields and pointers
https://github.com/flutter/flutter/issues/95541 - old iphones have huge problems with textfields

Hope this helps :)
